### ISSUE TITLE
Can't figure out how to run the script

### ISSUE THREAD

**Issue by diegovt21** [fedoco/itc-reporter#19]
Hi there!

I'm trying to use the script on a Windows laptop. I already have an Access Token but I just don't get what I have to do now. I've read the whole README.md and it talks about the Keychain, which support is limited to macOS.

Is there any way to make this work on Windows?

**Comment by fedoco** [comment]
Windows doesn't have a Keychain so you would have to remove all references to it in the code.

**Comment by diegovt21** [comment]
Alright, I think I removed every reference to it. Now I have to edit the parse_arguments func. with my account's data to make it work right? I specified my id, password and access token but can't figure ou what else do I need to do

**Comment by fedoco** [comment]
No, you shouldn't have to edit the parse_arguments function: The script already provides the parameters `-T` and `-P` (mind the case!) for usage without Keychain. If you're still having problems you need to explain them in greater detail, please.

**Comment by diegovt21** [comment]
Alright, can you explain me how to use it then? I don't know if I have to reference it from another script nor where or when I pass the credentials

**Comment by fedoco** [comment]
Well, I don't know what to explain besides what is already described in the README.md. But I'm confident you'll learn how to use it by yourself after you took a look at Apple's original Reporter tool! You can find the reference documentation at https://help.apple.com/itc/appsreporterguide/

**Comment by diegovt21** [comment]
Thank you! That clarifies it a lot. I was reading the README.md again and under _"Usage Examples"_ there's this code:
```sh
./reporter.py -u your@apple-id.com getStatus Finance -t "iTC Access Token"
```
Is that the exact command I have to be running on the terminal? Obviously with my Apple ID and in this case I would have to use -T since I'm not using Keychain, right?

**Comment by fedoco** [comment]
Exactly. This command with a `-T` instead of `-t` would tell you the status of Apple's endpoint and simultaneously confirms that your Access Token is working.

**Comment by diegovt21** [comment]
Awesome! It's working, thank you very much!
Is it possible to save the reports into .csv files?

**Comment by fedoco** [comment]
Reports are saved into tab-separated files automatically.
